hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/OMConfigKeys.java

```diff
@@ -292,4 +292,8 @@ private OMConfigKeys() {
   public static final long OZONE_OM_ADMIN_PROTOCOL_WAIT_BETWEEN_RETRIES_DEFAULT
       = 1000;
 
+  public static final String OZONE_OM_MAX_PENDING_REQ_COUNT =
+      "ozone.om.max.pending.req.count";
```

Comment:
ozone.om.max.pending.req.count gives me the idea that it is max pending queue size client requests to OM. Can we rename this to something like ozone.om.double.buffer.max.size or ozone.om.unflushed.transactions.max.size or something to denote it is for the OM DoubleBuffer.